ActionsRouter.js

```diff
@@ -0,0 +1,67 @@
+const express = require('express')
+const actionModel = require('./data/helpers/actionModel')
+
+const router = express.Router();
+
+router.get('/', async (req, res) => {
+    try {
+        const actions = await actionModel.get();
+        res.status(200).json(actions);
+    } catch(error) {
+        res.status(404).json({ message: error})
```

Comment:
the .catch() is meant to server errors, 404 is a user error, we usually use 500 status code inside of .catch(), the 404 would have been better inside of the check you are missing above.